### ISSUE TITLE
Data

### ISSUE THREAD

**Issue by liyajing293-byte** [RollingPlain/SAGE_IVIF#11]
Hello, I would like to ask how the dataset used in the article is divided into the training set and test set. I could not find any related explanation in the official documentation. Thank you for your help!
